src/domain/chainContext.ts

```diff
@@ -232,14 +233,12 @@ export class ChainContext {
       await asyncSleep(WATCHDOG_FREQUENCY);
       const currentTime = new Date().getTime();
       const timeElapsed = currentTime - timeLastBlockProcessed;
-      // Todo: set this as trace to avoid spamming the logs
-      console.log(
-        `[runBlockWatcher:chainId:${chainId}] diff: ${timeElapsed}ms`
-      );
+
+      log.trace(`diff: ${timeElapsed}ms`);
```

Comment:
nit: Can you add more context to the log?

Reply:
Done